### ISSUE TITLE
Question about get_mask_from_c2w() in Mapper.py file

### ISSUE THREAD

**Issue by zhw-github** [cvg/nice-slam#55]
Hi:
Very great work.But there are some things I don't understand about the get_mask_from_c2w() class methods in the Mapper.py file
this is great work.
1.125 row: cam_cord[:, 0] *= -1 why multiply by -1.
2.127 row: z = uv[:, -1:]+1e-5 why add 1e-5
3. 149 row:mask = mask & (0 <= -z[:, :, 0]) & (-z[:, :, 0] <= depths+0.5, why use -z,not z.And i don't understand this code.
4. 157,158rows: dist = torch.sum(dist*dist, axis=1)
        mask2 = dist < 0.5*0.5 , why use 0.5?And What do these two codes mean?

**Comment by Zzh2000** [comment]
Hi!
1,3 is because nerf's XYZ axis convention is different than the projection XYZ axis. 2 is for avoiding overflow.  4 is mainly a hyperparameter, features near the camera should be optimized otherwise it is possible that some rays are always occluded because the features right in front of the camera are not optimized (features not optimized will be decoded as occupied).
